src/mmg3d/mmg3d1_delone.c

```diff
@@ -1141,19 +1141,16 @@ int MMG5_mmg3d1_delone(MMG5_pMesh mesh,MMG5_pSol met,MMG5_int *permNodGlob) {
     return 0;
   }
 
-  int ier = 1;
-
   if ( mesh->info.iso && !MMG3D_chkmani(mesh) ) {
-    fprintf(stderr,"\n  ## Non orientable implicit surface. Exit program.\n");
-    ier = 0;
+    fprintf(stdout,"\n  ## WARNING : Non orientable implicit surface.\n");
```

Comment:
Please, change the `WARNING :` by a `Warning: %s:`.

While Mmg application and library files raise warning and error about inputs in upper case, all other messages (internal to mesh modification process) are formated in the same way (`## Warning: <func_name>` for warnings.)